addons/website/static/src/components/wysiwyg_adapter/wysiwyg_adapter.js

```diff
@@ -0,0 +1,98 @@
+/** @odoo-module */
+
+import { ComponentAdapter } from 'web.OwlCompatibility';
+import { _t } from '@web/core/l10n/translation';
+
+const { onWillStart, onMounted } = owl;
+
+
+export class WysiwygAdapterComponent extends ComponentAdapter {
+    /**
+     * @override
+     */
+    setup() {
+        const options = this.props.options || {};
+        this.iframe = this.props.iframe;
+
+        onWillStart(() => {
+            this.editable.classList.add('o_editable');
+            this.editableFromEditorMenu(this.$editable).addClass('o_editable');
+        });
+
+        onMounted(() => {
+            this.props.websiteContext.edition = 'started';
```

Comment:
I would avoid mutating props, and get the websiteContext from the service directly (I don't know if there was an issue at the time but now there is a new hook `useWowlService`)